docs/user_guide/labels_and_annotations_in_yunikorn.md

```diff
@@ -44,5 +44,10 @@ All annotations are under the namespace `yunikorn.apache.org`. For example `yuni
 | `task-groups`                | Defines the set of task groups for this application for gang scheduling. Each pod within an application must define all task groups.                                                   |
 | `schedulingPolicyParameters` | Arbitrary key-value pairs used to customize scheduling policies such as gang scheduling.                                                                                               |
 | `placeholder`                | Set if this pod represents a placeholder for gang scheduling. Set internally by YuniKorn.                                                                                              |
+| `allow-preemption` | `yunikorn.apache.org/allow-preemption: false` marks the pod with Opt out (OP).<br/> When using preemptor, pods marked with OP be used to choose victims as a last option.<br/> More informations are in the [DaemonSet Scheduling using Simple Preemptor](./../design/simple_preemptor).  |
```

Comment:
no line breaks in the text (remove <br\/>)
The `allow-preemption` annotation can only be set on a PriorityClass object. It will trigger opt out of preemption for pods with that specific priority class. Can point to the design doc to explain.